msg.go

```diff
@@ -921,7 +921,7 @@ func packStructValue(val reflect.Value, msg []byte, off int, compression map[str
 				copy(msg[off:off+len(s)], s)
 				off += len(s)
 			case `dns:"octet"`:
-				bytesTmp := make([]byte, 0)
+				bytesTmp := make([]byte, 256*4+1)
```

Comment:
it is the size we expect before we need to enlarge it. What the average binary size for CAA here?